terraform/YJAF-POC-APP.tf

```diff
@@ -52,5 +52,25 @@ module "YJAF-POC-APP" {
       added_by     = "Jon Dent jon.dent@justice.gov.uk"
       review_after = "2022-12-31"
     },
+    {
+      github_user  = "InFlamesForever"
+      permission   = "admin"
+      name         = "Richard Came"
+      email        = "richard.came@necsws.com"
+      org          = "NEC Software Solutions"
+      reason       = "YJAF developer. Needing access to the work on BackEnd coding Tasks"
+      added_by     = "Gareth Davies <gareth.davies@digital.justice.gov.uk"
+      review_after = "2022-08-27"
+    },
+    {
+      github_user  = "richardbradwell"
+      permission   = "admin"
+      name         = "Richard Bradwell"
+      email        = "richard.bradwell@necsws.com"
+      org          = "NEC Software Solutions"
+      reason       = "YJAF developer. Needing access to the work on BackEnd coding Tasks"
+      added_by     = "Jo Harvey <joanna.harvey@necsws.com"
```

Comment:
I think this line will fail as it not a justice email

Reply:
And it did. I'll go in and change update them. Should just be for the new additions.

Reply:
added_by values updated.